src/model/model.ts

```diff
@@ -36,6 +36,8 @@ const BuffSchema = z.object({
     dc: DiceFormulaSchema.optional(),
     save: DiceFormulaSchema.optional(),
     condition: CreatureConditionSchema.optional(),
+    applyDamage: DiceFormulaSchema.optional(),
+    halfOnSave: z.boolean().optional(), //TODO I need to combine the applyDamage and halfOnSave to a single... object?
```

Comment:
No big preferences here, but one of the requirements I have for this project is that updates must be backwards compatible with whatever the users have in their localStorage. So if you're going to wrap these two properties in an object later, might as well do it before this PR gets merged :+1: